### DISCUSSION TITLE
[FEATURE] Switch with two labels on each side

### DISCUSSION THREAD

**Discussion by cavasinf** [tabler/tabler#1223] [Ideas]
**Is your feature request related to a problem? Please describe.**
When using switches with detailed state like:
- Private / Public 
- And not simply: Public ? Off / On.

We need to override `.form-check-input` margin when using `.row` and `.col-auto`

**Describe the solution you'd like**
More easy way to implement switches with two labels.

**Describe alternatives you've considered**
Using row and style attribute override:

![image](https://user-images.githubusercontent.com/25293190/148045303-8e0c94cd-6d72-4b2c-9f0b-a33b61f180cb.png)
```html
<label class="row">
   <span class="col-auto">Private</span>
   <span class="col-auto">
     <label class="form-switch">
        <input type="checkbox" style="margin-left: -2.1rem;" class="form-check-input">
     </label>
   </span>
   <span class="col-auto">Public</span>
</label>
```

**Comment by kikikiblue** [comment]
👍
